src/frame/headers.rs

```diff
@@ -343,6 +342,17 @@ impl<T> From<Headers> for Frame<T> {
     }
 }
 
+impl fmt::Debug for Headers {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_struct("Headers")
+            .field("stream_id", &self.stream_id)
+            .field("stream_dep", &self.stream_dep)
+            .field("flags", &self.flags)
+            // `fields` and `pseudo` purposefully not included
```

Comment:
I think it's probably fine to not dump headers at the server log level... dunno if @olix0r has any thoughts on the matter.